src/tezos_interop/tezos_bridge.ml

```diff
@@ -93,10 +116,22 @@ type t = Long_lived_js_process.t
 let spawn () =
   let file = Scripts.file_tezos_js_bridge in
   Long_lived_js_process.spawn ~file
-let transaction t ~rpc_node ~secret ~required_confirmations ~destination
+let listen_transaction t ~rpc_node ~required_confirmations ~destination =
```

Comment:
```suggestion

let listen_transaction t ~rpc_node ~required_confirmations ~destination =
```
To keep consistent with the rest of the file we should have a newline separating the functions. We need a linting rule to standardize this - I don't care one way or the other, but it's confusing when it switches.

Reply:
There is no standard, the switch in many places are intentional, this was one missing